tests/e2e/backup_restore_suite_test.go

```diff
@@ -312,6 +213,7 @@ var _ = Describe("AWS backup restore tests", func() {
 			Expect(succeeded).To(Equal(true))
 			log.Printf("Backup for case %s succeeded", brCase.Name)
 
+			// TODO DataMover as well?
```

Comment:
From what I tested, volumesnapshot are deleted after backup/restore for datamover, so can I let just CSI here?

Reply:
> volumesnapshot are deleted after backup/restore for datamover

If that's the case, and backup only completes after volumesnapshot is gone for velero data mover then this would just be CSI here.